src/components/overlays/Dialog/DialogTrigger.tsx

```diff
@@ -41,6 +41,7 @@ export interface CubeDialogTriggerProps {
   mobileViewport?: number;
   /** The style map for the overlay **/
   styles?: Styles;
+  closeBehavior?: 'hide' | 'remove';
```

Comment:
I'm pretty sure it should be called `destroy`, not `remove`.
<img width="790" alt="image" src="https://user-images.githubusercontent.com/327209/213213174-810c2a1b-b280-40cf-a382-3b6cfcad8fb9.png">

Reply:
@tenphi I've read the answer from `ChatGPT` and it looks like it stands on my side ))

Btw: https://developer.mozilla.org/en-US/docs/Web/API/Element/remove, using `remove` matches with web platform better than `destroy`

Reply:
Renamed `hideBehavior` -> `hideOnClose`